common/scripts/base.sh

```diff
@@ -99,8 +99,7 @@ function undeploy() {
 # $2 - service port
 function waitUntilServiceIsActive() {
   marathon_lb_host_ip=$(dcos marathon app show marathon-lb | $mesos_artifacts_home/common/scripts/get-marathon-lb-host.py)
```

Comment:
@ravihansa3000: Used the host ip to check the service status for all the services in commit 71584056cd44815e491ec4484c5dfc15a903bcd2. Added logic to wait until the host ip is returned by get-host-ip python script. Please give your suggestions.

@isurulucky: +1 to use marathon lb's hostname for mgmt console urls.